components/raftstore/src/store/worker/metrics.rs

```diff
@@ -114,7 +114,7 @@ lazy_static! {
     .unwrap();
     pub static ref LOCAL_READ_EXECUTED_CACHE_REQUESTS: IntCounter = register_int_counter!(
         "tikv_raftstore_local_read_cache_requests",
-        "Total number of requests directly executed by local reader."
+        "Total number of cache hits by local reader."
```

Comment:
```suggestion
        "Total number of snapshot cache hits by local reader."
```
Better to clarify it's "snapshot cache". wdyt @Little-Wallace